lib/handler/console.js

```diff
@@ -40,7 +40,7 @@
       }
 
       console.error(logValues);
-    } else {
+    } else if (typeof logAttributes.msg !== 'undefined') {
```

Comment:
I'll take a look at this tomorrow, thanks for the comment! I didn't expect a response this evening.